src/interceptors/ClientRequest/utils/recordRawHeaders.ts

```diff
@@ -118,7 +118,7 @@ export function recordRawFetchHeaders() {
             [Reflect.get(headersInit, kRawHeaders)],
             newTarget
           )
-          ensureRawHeadersSymbol(headers, Reflect.get(headersInit, kRawHeaders))
+          ensureRawHeadersSymbol(headers, [...Reflect.get(headersInit, kRawHeaders)])
```

Comment:
It would be great to add a `@note` comment here explaining that spreading the headers creates their copy + what issue this prevents.